plugins/org.jkiss.dbeaver.ext.altibase.ui/src/org/jkiss/dbeaver/ext/altibase/ui/internal/AltibaseUIMessages.properties

```diff
@@ -0,0 +1,22 @@
+
+edit_altibase_typeset_manager_container = Container
+
+edit_altibase_typeset_manager_dialog_title = Create typeset
```

Comment:
```suggestion
edit_altibase_typeset_manager_dialog_title = Create Typeset
```

Reply:
Okay, for separated words, the second letter will also be capitalized.